constants.py

```diff
@@ -143,6 +143,10 @@
 supportedPlatforms["10.10"] += supportedPlatforms["10.9"]
 
 supportedPlatforms["10.11"] = supportedPlatforms["10.10"]
+supportedPlatforms["10.11"] += [
+        'aarch64-ubuntu-2304',
+        'amd64-ubuntu-2304',
+        ]
```

Comment:
will only 10.11+ support ubuntu-2304?

Reply:
yes, plan on getting 10.11 (or 11.0) into this ubuntu release so we don't need to support earlier.